src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/reporting/ReportingImporterTest.kt

```diff
@@ -15,7 +15,7 @@ import java.time.LocalDate
 @ExtendWith(SpringExtension::class)
 @Import(TestConfig::class)
 @ActiveProfiles("test")
-//@ActiveProfiles("localstack")
+//@ActiveProfiles("dev")
```

Comment:
Is the commented out stuff supposed to be here?

Reply:
Yeah...still switching between different profiles when testing. Maybe I could have conditional logic, I'll look at that